x/slashing/keeper/infractions.go

```diff
@@ -12,7 +13,7 @@ import (
 )
 
 // HandleValidatorSignature handles a validator signature, must be called once per validator per block.
-func (k Keeper) HandleValidatorSignature(ctx sdk.Context, addr cryptotypes.Address, power int64, signed bool) {
+func (k Keeper) HandleValidatorSignature(ctx sdk.Context, addr cryptotypes.Address, power int64, signed comet.BlockIDFlag) {
```

Comment:
How do we go about changelog and upgrading.md? These are missing, can for sure definitely be done a follow-up, but we should have a tracking issue for this if not done here.
We really need to only put changes in the changelog that has been released as well, as a few never released API are changed as well.

Reply:
added this to the tracking issue https://github.com/cosmos/cosmos-sdk/issues/12272